argo/src/Argo.hs

```diff
@@ -184,6 +186,10 @@ data JSONRPCException =
     , errorID :: Maybe RequestID
       -- ^ The request ID, if one is available. @Nothing@ omits
       -- it, because JSON-RPC defines a meaning for null.
+    , errorStdOut :: String
```

Comment:
I wonder if this should be `Maybe Text`.

My thought process is that `Text` is consistent with our other string types, and a quick `T.pack` on the stdout seems unlikely to make things appreciably slower (especially because we would need to traverse the whole thing anyway to send it over the wire). `Maybe` lets us distinguish between things that did not do stdio (e.g. by returning it as a JSON null). OTOH, it would never be `Just ""` (would it?), so this might be silly.